src/Parser/Core/Modules/Items/GarothiFeedbackConduit.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+
+import ITEMS from 'common/ITEMS';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SPELLS from "../../../../common/SPELLS";
+import { formatThousands } from "../../../../common/format";
```

Comment:
Use absolute path for stuff in common / Core

Reply:
Woops, my IDE seems to auto add them like that :/ fixed